addons/hr_expense/models/hr_expense.py

```diff
@@ -256,12 +256,14 @@ def _compute_product_description(self):
     @api.depends('product_id', 'company_id')
     def _compute_unit_amount(self):
         for expense in self:
-            if not expense.product_id or not expense.product_has_cost or expense.attachment_number or (not expense.attachment_number and expense.unit_amount):
+            product = expense.product_id
+            if not product or not expense.product_has_cost or expense.attachment_number or (not expense.attachment_number and expense.unit_amount):
                 continue
-            expense.unit_amount = expense.product_id.price_compute(
+            expense.unit_amount = product.price_compute(
                 'standard_price',
                 uom=expense.product_uom_id,
-                currency=expense.currency_id)[expense.product_id.id]
+                company=expense.company_id
```

Comment:
Why are you removing the currency ? What if the currency of the expense is different from the currency of the company ? Should we keep both ? Idk just wondering ^^

Reply:
Because it rounds to the currency decimal (2) and not the Product Price decimal (2 by default but if we set it to 3 we want to keep it to 3) THEN rounding it to the product price on the view ending in things like 0.005 showing 0.010